spec/text-editor-component-spec.coffee

```diff
@@ -68,48 +68,111 @@ describe "TextEditorComponent", ->
       expect(nextAnimationFrame).not.toThrow()
 
   describe "line rendering", ->
-    it "renders the currently-visible lines plus the overdraw margin", ->
-      wrapperNode.style.height = 4.5 * lineHeightInPixels + 'px'
+    expectLineRender = (tileNode, {screenRow, top}) ->
```

Comment:
This is minor, but I am not very happy with how this test helper is named. Probably this would look better with a matcher?

```coffee
expect(tilesNodes[0]).toHaveRenderedLine(screenRow: 0, top: 10)
```